docs/usage/quickstart.rst

```diff
@@ -91,24 +125,24 @@ run)::
     Launching MetalK8s shell environment. Run 'exit' to quit.
     (metal-k8s) $
 
-Now we're all set to deploy a cluster::
+Now, you're all set to deploy a cluster::
 
     (metal-k8s) $ ansible-playbook -i inventory/quickstart-cluster -b playbooks/deploy.yml
 
 Grab a coffee and wait for deployment to end.
 
-Inspecting the cluster
-----------------------
-Once deployment finished, a file containing credentials to access the cluster is
-created: :file:`inventory/quickstart-cluster/artifacts/admin.conf`. We can
-export this location in the shell such that the :program:`kubectl` and
-:program:`helm` tools know how to contact the cluster *kube-master* nodes, and
-authenticate properly::
+Inspect the Cluster
+-------------------
+
+Once deployment finishes, a file containing credentials to access the cluster is
+created: :file:`inventory/quickstart-cluster/artifacts/admin.conf`. Export this
+location in the shell so that the :program:`kubectl` and :program:`helm` tools
+know how to contact the cluster *kube-master* nodes, and authenticate properly::
 
     (metal-k8s) $ export KUBECONFIG=`pwd`/inventory/quickstart-cluster/artifacts/admin.conf
 
-Now, assuming port *6443* on the first *kube-master* node is reachable from your
-system, we can e.g. list the nodes::
+Assuming port *6443* on the first *kube-master* node can be reached from your
```

Comment:
As general comment, for values/variables that are used in the deployment, using `this format` instead of *this format* or **this format** makes it much more readable, in my opinion. In this doc, this would pertain to terms such as `kube-node`, `kube-master`, `PersistentVolumes`, and various code commands that are imbedded in text paragraphs.

Reply:
It's primarily about assigning the correct roles to terms, not really about formatting :)

Reply:
From a reader's POV, using all 3 kinds of formatting looks arbitrary and lacks unity across the document. Unless I missed it, it doesn't seem that this document includes explanations on what these 'roles' are either. The section where the diverse formatting is most confusing exists between lines 83 and 116. 

I realize the roles are visible in the `.rst` file, but again, from a user POV, this is not clear or visible.

Unless these roles are linked to additional functionality, I must insist a standard way of formatting technical terms. Specificity is not useful when a first time user is simply looking for a quick and dirty start.

Reply:
Oh, I agree. Thing is, currently no roles are used at all for these terms, group names,... so indeed, the formatting is ad-hoc and likely inconsistent.

When a role is applied, the intent is indeed for those terms, group names,... to be formatted differently though consistently, in the rendered result. See e.g. how file-names, program names and inline commands are already formatted in https://metal-k8s.readthedocs.io/en/latest/usage/quickstart.html

Reply:
I see it, and that's why I'm raising this issue because I disagree with the way it is already formatted/will be formatted.

The detailed formatting will not be picked up by first-time users/readers, because the 'roles' that are assigned in the `.rst` file *are not reflected in the finalized document*.

If you are adamant that this quick-start guide have such detailed formatting (detailed, but unclear), then there **needs** to be a section at the top explaining what each formatting method means. Otherwise, use `this kind of formatting` for all technical terms, names, and values.